skycoin/skywire
# Feature/host keeper

Did you run `make format && make check`? Yes

Fixes [skywire-services#504](https://github.com/SkycoinPro/skywire-services/issues/504)

 Changes:	
- add an update host information routing on starting visor.

How to test this PR:
Check skywire-service related PR [here](https://github.com/SkycoinPro/skywire-services/pull/511).
